lib/presentation/widget/myspace/my_space_widget.dart

```diff
@@ -136,11 +136,13 @@ class _MySpaceWidgetState extends State<MySpaceWidget> {
           if (documentList.isEmpty) {
             return _buildUploadFileHere(context);
           } else {
-            return ListView.builder(
+            return RefreshIndicator(
+              onRefresh: () async => await mySpaceViewModel.getAllDocument(),
```

Comment:
nice, but we don't need to await here:
```onRefresh: () async => mySpaceViewModel.getAllDocument(),
```

Reply:
Why not call on a thunk action on refresh?

Reply:
I want it consistent for all the app. if we can. Please do that. if not, please explain to me. My poor in Widget :(

Reply:
since `onRefresh ` waits for `Future<void>` I have to make this async I think

> I want it consistent for all the app. if we can. Please do that. if not, please explain to me. My poor in Widget :(

I think we'll encounter 2 problems : Weird behaviour on swipe refresh, gesture will be too high on the screen, binding onRefresh method will be tough

Reply:
okie, my false. Move on with: 
```
onRefresh: () async => mySpaceViewModel.getAllDocument(),
```
Is it ok?

Reply:
Ok I'll call action too

Reply:
> Ok I'll call action too

only remove `await`. That right?

Reply:
oh yes forgot it was already an action lol

Reply:
Mb for 

> Weird behaviour on swipe refresh, gesture will be too high on the screen

It can be custom in parameter